.php_cs.dist

```diff
@@ -9,7 +9,7 @@ return PhpCsFixer\Config::create()
         '@Symfony' => true,
         '@Symfony:risky' => true,
         '@PHPUnit75Migration:risky' => true,
-        'php_unit_dedicate_assert' => ['target' => '3.5'],
+        'php_unit_dedicate_assert' => ['target' => '5.6'],
```

Comment:
I'm not sure we should do this, as it will make code in 3.4 diverge from 4.x, thus making merges conflict.

Reply:
can't we use this in branch 3.4, with a proper polyfill?

Reply:
rebase in 3.4. 

Require #32931